models/consistency.go

```diff
@@ -246,3 +247,23 @@ func FixIssueLabelWithOutsideLabels() (int64, error) {
 
 	return res.RowsAffected()
 }
+
+// CountActionCreatedUnixString count created_unix that are a string
+func CountActionCreatedUnixString() (int64, error) {
```

Comment:
Does this count actions with a created_unix of a string type(everything is string in SQLite) or when it's empty. Seems like this comment isn't correct.

Reply:
only when it's an empty string, indeed